packages/data-stores/src/launch/reducer.ts

```diff
@@ -57,21 +57,6 @@ const plan: Reducer< Plans.Plan | undefined, LaunchAction > = ( state, action )
 	return state;
 };
 
-/**
- * To keep track of the last paid plan the user has picked
- * This is useful information as this paid plan can be suggested later
- *
- * @param state the state
- * @param action the action
- */
-const paidPlan: Reducer< Plans.Plan | undefined, LaunchAction > = ( state, action ) => {
-	if ( action.type === 'SET_PLAN' && ! action.plan?.isFree ) {
-		return action.plan;
-	}
-	return state;
-};
-
```

Comment:
We're not using this reducer anywhere so let's remove it for now.

Reviewed as part of #47993